adb_graphics/conversions.py

```diff
@@ -7,91 +7,91 @@
 
 import numpy as np
 
-def k_to_c(field: np.ndarray, **kwargs) -> np.ndarray:
```

Comment:
@cshartsough brought up the issue of some variables being numpy arrays, while others were xarrays. I've removed all instances I could find of converting an array to a numpy array.